### PR TITLE
UAR-594: Changes for Ind Name and Contact

### PR DESCRIPTION
**Pull request by jkayyala**
### JIRA link

https://companieshouse.atlassian.net/browse/UAR-594

### Change description
Change format for Ind MO name during comparison and ignoreCase for contact name and email

Response from public MO endpoint
image

Response from private MO endpoint
image

### Work checklist

- [x] Tests added where applicable

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by bantao-ch — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/ManagingOfficerChangeService.java [hunk 1]
```diff
@@ -167,7 +167,7 @@ private ManagingOfficerChange<IndividualManagingOfficer> convertManagingOfficerI
                 personName,
                 CompanyOfficerApi::getName,
                 Function.identity(),
-                ComparisonHelper::equals,
+                ComparisonHelper::equalsIndividualMO,
```
Comment (AndreKramer7654) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/ManagingOfficerChangeService.java:170]: equalsIndividualMoName

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java [hunk 2]
```diff
@@ -139,6 +139,15 @@ public static boolean equals(PersonName personName, String string) {
         return StringUtils.equalsIgnoreCase(normalise(personName.toString()),normalise(string));
     }
 
+    public static boolean equalsIndividualMO(PersonName personName, String string) {
+        var nullValuesCheck = handleNulls(personName, string);
+        if (nullValuesCheck != null) {
+            return nullValuesCheck;
+        }
+        String individualMoFormat = personName.getSurname().concat(", ").concat(personName.getForename());
```
Comment (AndreKramer7654) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java:147]: Does the UI include a space? We normalise() but that would still be ", "

Comment (jkayyala) [reply src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java:147]: seems its done in https://github.com/companieshouse/api.ch.gov.uk/blob/40479583de693e536c3a05bd50d54e2e8920965b/lib/ApiChGovUk/Controller/CompanyOfficers.pm#L381C8-L384C1

 # Only join surname and forenames if the officer has forenames
        $officer->{name} = join ', ', ($officer->{name}, join ' ', map { $appts_officer->{$_} } @forenames)
            if scalar @forenames;

Comment (AndreKramer7654) [reply src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java:147]: should be ", "

**Review by AndreKramer7654 — APPROVED** [review]

**Pull request merged**